### PR TITLE
Enable GHA for the tarball-client repo (SOFTWARE-4359)

### PR DESCRIPTION
**Pull request by yongleyuan**

### CODE REVIEW

File: .github/workflows/ci-tests.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      OSG_VERSION: ${{ matrix.OSG_VERSION }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Before Installation
+        run: |
+          sudo apt-get update
+          echo 'DOCKER_OPTS="-H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker > /dev/null
+          sudo service docker restart
+          sleep 5
+          sudo docker pull centos:centos${OS_VERSION}
```
Comment (brianhlin) [inline .github/workflows/ci-tests.yml]: I think this may be unnecessary

**Review by brianhlin — COMMENTED** [review]
@yongleyuan go ahead and turn this into a regular PR. Also, this action should have run on your local fork: do you have a link to the latest run?

**Comment by yongleyuan** [comment]
> @yongleyuan go ahead and turn this into a regular PR. Also, this action should have run on your local fork: do you have a link to the latest run?

Here it is: https://github.com/yongleyuan/tarball-client/actions/runs/1047942793

File: .github/workflows/ci-tests.yml (hunk 1 above)
Comment (yongleyuan) [reply .github/workflows/ci-tests.yml]: The whole `Before Installation` step is unnecessary? (Just making sure)

Comment (brianhlin) [reply .github/workflows/ci-tests.yml]: Yeah, I think so. Try removing it and seeing if the action in your own repo works. If it does, then we can fully drop this step in the PR.

File: .github/workflows/ci-tests.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: CI Tests
```
Comment (brianhlin) [inline .github/workflows/ci-tests.yml]:
```suggestion
name: Test tarball builds
```

CI tests is pretty generic (we may add different CI tests later!)

File: .github/workflows/ci-tests.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  ci-test:
```
Comment (brianhlin) [inline .github/workflows/ci-tests.yml]:
```suggestion
  test-build:
```

Again, a more specific name

**Review by brianhlin — CHANGES_REQUESTED** [review]
Minor naming changes so that the test results are a little clearer

**Comment by yongleyuan** [comment]
> Minor naming changes so that the test results are a little clearer

Do I need to change the file name?

**Comment by brianhlin** [comment]
> Do I need to change the file name?

Yes, please!

File: .github/workflows/ci-tests.yml (hunk 1 above)
Comment (yongleyuan) [reply .github/workflows/ci-tests.yml]: I tried removing `Before Installation` step, and the [workflow](https://github.com/yongleyuan/tarball-client/actions/runs/1068897669) ran successfully.

**Review by brianhlin — APPROVED** [review]
LGTM!

**Pull request merged**